src/Core/Managed/Shared/Extensibility/Metrics/MetricTerms.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace Microsoft.ApplicationInsights.Extensibility.Metrics
+{
+    internal static class MetricTerms
+    {
+        private const string MetricPropertiesNamePrefix = "Microsoft.ApplicationInsights.Metrics";
+        public static class Extraction
+        {
+            //Microsoft.ApplicationInsights.Metrics.MetricIsAutocollected
+            public static class ConsideredByProcessors
+            {
+                public static class Moniker
```

Comment:
Why do you have 4 nested classes pretending to be a namespace for a single const string?